3529/2/telegram.rb

```diff
@@ -0,0 +1,30 @@
+require 'telegram/bot'
+require 'yaml'
+require 'httparty'
+require 'open-uri'
+require_relative 'start_context'
```

Comment:
You can require all files with 1 line

Reply:
it's about require_all? so i need to put all files i use in a one dir, am i right?

Reply:
1 way: 
https://coderwall.com/p/sfokrw/require-all-files-within-a-directory-in-ruby
Find the best solution for your project

Reply:
thanks)))